source/acquaman/BioXAS/BioXASXASScanActionController.cpp

```diff
@@ -158,6 +174,8 @@ void BioXASXASScanActionController::buildScanControllerImplementation()
 		connect( BioXASBeamline::bioXAS()->i0Detector(), SIGNAL(darkCurrentValueChanged(double)), i0CorrectedDetectorSource, SLOT(setDarkCurrent(double)) );
 
 		scan_->addAnalyzedDataSource(i0CorrectedDetectorSource, true, false);
+
+		exportXDI->addDataSource(i0CorrectedDetectorSource->name(), false);
```

Comment:
Just making sure... you did want the omit axis value argument for this one to be false, but all the others true?